man/journald.conf.xml

```diff
@@ -358,6 +358,13 @@
       </varlistentry>
 
       <varlistentry>
+        <term><varname>ReadKmsg=</varname></term>
```

Comment:
the other kmsg-related options use the `KMsg` casing, this one should too... (see MaxLevelKMsg=, ForwardToKMsg=)